bot/exts/evergreen/conversationstarters.py

```diff
@@ -0,0 +1,65 @@
+import json
+import random
+from pathlib import Path
+
+from discord import Embed
+from discord.ext import commands
+
+from bot.constants import WHITELISTED_CHANNELS
+from bot.utils.decorators import override_in_channel
+
+
+with Path("bot/resources/evergreen/starter.json").open("r", encoding="utf8") as f:
+    STARTERS = json.load(f)["starters"]
+
+
+with Path("bot/resources/evergreen/py_topics.json").open("r", encoding="utf8") as f:
+    # First ID is #python-general and the rest are top to bottom categories of Topical Chat/Help.
+    PY_TOPICS = json.load(f)["python-channels"]
+
+    # All the allowed channels that the ".topic" command is allowed to be executed in.
+    ALL_ALLOWED_CHANNELS = [int(channel_id) for channel_id in PY_TOPICS.keys()].extend(WHITELISTED_CHANNELS)
```

Comment:
`list.extend` returns `None`. I assume you meant something like:
```suggestion
    ALL_ALLOWED_CHANNELS = [int(channel_id) for channel_id in PY_TOPICS.keys()] + list(WHITELISTED_CHANNELS)
```